src/umpire/strategy/DynamicPool.hpp

```diff
@@ -99,7 +102,7 @@ class DynamicPool :
     std::size_t getReleasableSize() const noexcept;
```

Comment:
This method was measuring the total size of the free blocks before, even though some of those blocks were not the head of allocations and could not be released back to the resource. Thus, some of the strategy tests had to change.